tests/integration/test_chat.py

```diff
@@ -251,6 +251,12 @@
         - Please repeat: step2
 """
 
+yaml_notice = """
+- 1:
+    notice-type: {}
```

Comment:
Added the placeholder. Also changed `notice-text` to hold a placeholder, to make it simpler to write tests.